### PR TITLE
chore: remove params from journey/start event payload 

### PR DESCRIPTION
**Pull request by shahzad31**
We are pushing params values to journey start document payload. It serves no purpose. So in this PR i am removing those.

### CODE REVIEW

**Review by vigneshshanmugam — CHANGES_REQUESTED** [review]
It was intentional and it requires a UI modifications to take full advantage. See the issue here - https://github.com/elastic/synthetics-dev/issues/62

File: __tests__/cli.test.ts [hunk 1]
```diff
@@ -152,13 +152,9 @@ describe('CLI', () => {
         return getJourneyStart(cli.output());
       };
 
-      expect((await output()).payload).toMatchObject({
-        params: { url: 'non-dev' },
-      });
+      expect((await output()).payload).toMatchObject({});
```
Comment (vigneshshanmugam) [inline __tests__/cli.test.ts:155]: this defaults the purpose of this test itself. We want to know whether the passed parameters from CLI are being used in the journey, but this change effectively discards that check itself. 

I would change the test to use a custom reporter and look for params being used as it is.

**Review by vigneshshanmugam — CHANGES_REQUESTED** [review]
Except for the CLI tests, rest looks good

File: __tests__/cli.test.ts (hunk 1 above)
Comment (shahzad31) [reply __tests__/cli.test.ts]: i have tried to update the tests but ended up doing lot more than desired, i hope it;s ok :D

Comment (vigneshshanmugam) [reply __tests__/cli.test.ts]: Fixed the tests in a different way, other than using params.

**Review by vigneshshanmugam — APPROVED** [review]
LGTM

**Pull request merged**